src/index.html

```diff
@@ -17,11 +17,377 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="style.scss"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
     />
```

Comment:
You are importing the same Google Fonts twice (lines 14-17 and 23-34). This is unnecessary and can slow down your page load speed. You should remove the duplicate.